documentation/source/models_export_pose.md

```diff
@@ -0,0 +1,1738 @@
+      quant_dim = list(amax.shape).index(list(amax_sequeeze.shape)[0])
+    
+
+    ============= Diagnostic Run torch.onnx.export version 2.0.1+cu118 =============
+    verbose: False, log level: Level.ERROR
+    ======================= 0 NONE 0 NOTE 0 WARNING 0 ERROR ========================
+    
+    ============= Diagnostic Run torch.onnx.export version 2.0.1+cu118 =============
+    verbose: False, log level: Level.ERROR
+    ======================= 0 NONE 0 NOTE 0 WARNING 0 ERROR ========================
+    
+    
+
+
+    
+![png](models_export_pose_files/models_export_pose_37_2.png)
+    
+
+
+### Advanced INT-8 quantization options
+
+When quantizing a model using `quantization_mode==ExportQuantizationMode.INT8` you can pass a DataLoader to export() function to collect correct statistics of activations to prodice a more accurate quantized model.
+We expect the DataLoader to return either a tuple of tensors or a single tensor. In case a tuple of tensors is returned by data-loader the first element will be used as input image.
+You can use existing data-loaders from SG here as is.
+
+**Important notes**
+* A `calibration_loader` should use same image normalization parameters that were used during training.
+
+In the example below we use a dummy data-loader for sake of showing how to use this feature. You should use your own data-loader here.
+
+
+```python
+import torch
+from torch.utils.data import DataLoader
+from super_gradients.conversion import ExportQuantizationMode
+
+# THIS IS ONLY AN EXAMPLE. YOU SHOULD USE YOUR OWN DATA-LOADER HERE
+dummy_calibration_dataset = [torch.randn((3, 640, 640), dtype=torch.float32) for _ in range(32)]
+dummy_calibration_loader = DataLoader(dummy_calibration_dataset, batch_size=8, num_workers=0)
+# THIS IS ONLY AN EXAMPLE. YOU SHOULD USE YOUR OWN DATA-LOADER HERE
+
+export_result = model.export(
+    "yolo_nas_pose_s_int8_with_calibration.onnx",
+    confidence_threshold=0.5,
+    output_predictions_format=DetectionOutputFormatMode.FLAT_FORMAT,
+    quantization_mode=ExportQuantizationMode.INT8,
+    calibration_loader=dummy_calibration_loader
+)
+
+session = onnxruntime.InferenceSession(export_result.output,
+                                       providers=["CUDAExecutionProvider", "CPUExecutionProvider"])
+inputs = [o.name for o in session.get_inputs()]
+outputs = [o.name for o in session.get_outputs()]
+result = session.run(outputs, {inputs[0]: image_bchw})
+
+show_predictions_from_flat_format(image, result)
+```
+
+     25%|██████████████████████████████████████▊                                                                                                                    | 4/16 [00:12<00:37,  3.14s/it]
+    W1025 15:56:09.825534 23736 tensor_quantizer.py:174] Disable HistogramCalibrator
```

Comment:
can we drop this extremely long print?

Reply:
I think these warnings coming from pytorch_quantization package. 
There are few options but all looks dirty to me
1) Go and check whether we should address these warnings at all. (Long, out of the scope). And either fix or silence them inside  export() method
2) Globally set log level for ERROR in this notebook
3) `%%capture` in cell to just silence the output

Reply:
I re-run notebook and it went away (Probably thanks to the bug we have in loggers that causes all outputs to be muted in notebooks)